spec/public-api.yaml

```diff
@@ -1,283 +1,243 @@
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/PaginatedRuleList"
-          description: ""
-  /v1/actions/rules/{id}:
+  /v1/actions/detector/{detector_id}/rules:
     get:
-      operationId: Get rule
-      description: Retrieve a rule
+      operationId: List detector rules
+      description: List all rules for a detector
       parameters:
-        - in: path
-          name: id
-          schema:
-            type: integer
-          description: Get a rule by its ID.
-          required: true
+      - in: path
+        name: detector_id
+        schema:
+          type: string
+        required: true
       tags:
-        - rules
+      - actions
       security:
-        - ApiToken: []
+      - ApiToken: []
       responses:
-        "200":
+        '200':
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/Rule"
-          description: ""
-    delete:
-      operationId: Delete rule
-      description: Delete a rule
-      parameters:
-        - in: path
-          name: id
-          schema:
-            type: integer
-          description: Delete a rule by its ID.
-          required: true
-      tags:
-        - rules
-      security:
-        - ApiToken: []
-      responses:
-        "204":
-          description: ""
-  /v1/actions/detector/{detector_id}/rules:
+                $ref: '#/components/schemas/PaginatedRuleList'
+          description: ''
     post:
       operationId: Create rule
-      description: Create a new rule for a detector.
+      description: Create a new rule for a detector
       parameters:
-        - in: path
-          name: detector_id
-          schema:
-            type: string
-          description: Choose a detector by its ID.
-          required: true
+      - in: path
+        name: detector_id
+        schema:
+          type: string
+        required: true
       tags:
-        - rules
+      - actions
       requestBody:
         content:
           application/json:
             schema:
-              $ref: "#/components/schemas/RuleCreationInput"
+              $ref: '#/components/schemas/RuleRequest'
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: '#/components/schemas/RuleRequest'
+          multipart/form-data:
+            schema:
+              $ref: '#/components/schemas/RuleRequest'
         required: true
       security:
-        - ApiToken: []
+      - ApiToken: []
       responses:
-        "201":
+        '201':
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/Rule"
-          description: ""
-  /v1/notes:
+                $ref: '#/components/schemas/Rule'
+          description: ''
+  /v1/actions/rules:
     get:
-      operationId: get notes
-      description: Retrieve notes for a detector
+      operationId: List rules
```

Comment:
Which rules? All the rules for a user? Or a group?

Reply:
Users and groups should share the exact same set of rules, so I'm not sure if that should be differentiated